docs-new/docs/graphql/core/event-triggers/clean-up.mdx

```diff
@@ -71,13 +69,11 @@ AND (delivered = true OR error = true);
 
 ## Option 5: Clear everything
 
-<div class="admonition">
-
-Warning
+:::info Warning
```

Comment:
This can actually be `:::caution` Warning as the title suggests.

Applies for any other titles with Warning! Earlier, color customization was not avaibale, but now we have!